executor/common_linux.h

```diff
@@ -5215,3 +5215,64 @@ static long syz_80211_join_ibss(volatile long a0, volatile long a1, volatile lon
 }
 
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_clone || __NR_syz_clone3
+#if SYZ_EXECUTOR
+// The slow down multiplier is already taken into account.
+#define USLEEP_FORKED_CHILD (3 * syscall_timeout_ms)
+#else
+// It's impossible to use replacements here, so we just hardcode it.
+#define USLEEP_FORKED_CHILD (3 * /*{{{BASE_CALL_TIMEOUT_MS}}}*/ *1000)
+#endif
+
+#ifndef CLONE_VM
+#define CLONE_VM 0x00000100
```

Comment:
```
       CLONE_VM (since Linux 2.0)
```

Do we want to support versions older than 2.0? :)

Reply:
Deleted.